src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -71,13 +71,13 @@ public static Event[] getSampleEvents() {
         Person[] samplePersons = getSamplePersons(); // Reuse sample persons for the event participants
 
         return new Event[] {
-            new Event("Tech Conference 2024",
+            new Event("NUS Student Life Fair 2024",
```

Comment:
I would kindly like to enquire the purpose of renaming this?
There is nothing wrong but I would like to know if there's a specific case you're aiming to test!